### ISSUE TITLE
Blood Displays Damage Numbers

### ISSUE THREAD

**Issue by RetroGamer2023** [BLOODWOLF333/Brutal-Doom-Community-Expansion#416]
### Version
brutalv21.13.1.zip

### Environment
Windows 10 x64

### Port
GZdoom 4.10.0

### Description
When you shoot an enemy dead, every bit of blood splatter displays it's own damage number.

Vanilla Brutal Doom v21 works just fine using the mod "damnums_1.0.2"

I used a clean config (GZdoom defaults and mod defaults) with just these 2 mods loaded.

**Comment by BLOODWOLF333** [comment]
How is this a brutalv21.13.1 problem and not a problem with damnums_1.0.2?

**Comment by BLOODWOLF333** [comment]
It's impossible to make a mod be compatible with the hundreds of thousands of other doom mods out there, thats not how this old 1990's decorate and ACS code works.  Read the description for Beautiful Doom he explains it much better https://www.moddb.com/mods/beautiful-doom-6100.  I dont play GZDoom all i play is zandronum multiplayer which doesnt support ZScript like the damnums mod uses so this is not my problem.

**Comment by RetroGamer2023** [comment]
Like I said it's not an issue you will find in BrutalDoom vanilla. Neither is a it a problem in PB, not a problem in Brutal Doom Extended, and generally with any other mods I mess with. But like you said, you don't support  GZDoom so who cares right?

Here's your selling point on ModDB:

"...
This brutalv21.pk3 replacement has been rebuilt, tweaked, sanded, and polished to a mirror-finish over the course of almost four years. Every bug you encountered in the past has been fixed. All BrutalDoom-specific features have been fleshed-out and fully realized with the combined efforts of the BrutalDoom community's many modders while keeping compatibility with zandronum and multiplayer at the forefront.
..."

I have no idea what the BrutalDoom Community uses but I suspect many of them do use GZDoom. I thought you liked bug fixing and polishing?

**Comment by BLOODWOLF333** [comment]
Yeah, all the bugs and features inside of Brutal Doom Only have been fixed-up, an outside mod no longer working is not a Brutal Doom bug and not my concern.  I've already spent most of today trying to figure out why its showing that "1000" when blood actors hit the floor and I haven't been able to make it stop.  It's silly for me to try and conform this to some other addon that I don't even use.  If you care then you are the one to make them work together because this is a waste of my time.
